lib/redis/cluster.rb

```diff
@@ -238,6 +239,29 @@ def try_send(node, method_name, *args, retry_count: 3, &block)
       raise
     end
 
+    def _scan(command, &block)
+      input_cursor = command[1].to_i
```

Comment:
Maybe use `Integer` here? So that it raises if we get something that's not an integer.

Reply:
@byroot updated